alarm_trap_sender.cpp

```diff
@@ -157,7 +193,14 @@ void AlarmTrapSender::issue_alarm(const std::string& issuer, const std::string&
     {
       if (!AlarmFilter::get_instance().alarm_filtered(index, severity))
       {
+        alarmActiveTable_create_row((char*) "", AlarmTrapSender::alarm_time_issued(), alarm_counter, alarm_table_def);
         send_trap(alarm_table_def);
+        alarm_counter++;
+        //Reset the alarm counter incramentor if it reaches its max limit
```

Comment:
You don't need the word "incrementor" in this sentence - 'Reset the alarm counter' makes sense by itself - but you do want a space after the `//`.